Package.swift

```diff
@@ -8,7 +8,7 @@ let package = Package(
     ],
     dependencies: [
         // Console protocols, terminal, and commands
-        .Package(url: "https://github.com/vapor/console.git", Version(2,0,0, prereleaseIdentifiers: ["beta"])),
-        .Package(url: "https://github.com/vapor/json.git", Version(2,0,0, prereleaseIdentifiers: ["beta"]))
+        .Package(url: "https://github.com/vapor/console.git", majorVersion: 2),
+        .Package(url: "https://github.com/vapor/json.git", majorVersion: 2)
```

Comment:
I've been putting trailing commas, I think the argument is that it saves git diff lines when things are added, not sure if we want to do everywhere.